lib/linter-plus.coffee

```diff
@@ -32,6 +32,15 @@ class Linter
     @subscriptions.add atom.workspace.onDidChangeActivePaneItem =>
       @commands.lint()
 
+    ignoreTabUpdates = true
```

Comment:
Okay, this looks horrible. If you want to only fire event on change, use `atom.config.onDidChange`.